cicd/pipelines/op-release-notes.adoc

```diff
@@ -23,6 +23,8 @@ include::modules/op-tkn-pipelines-compatibility-support-matrix.adoc[leveloffset=
 include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
+// include::modules/op-release-notes-1-8.adoc[leveloffset=+1]
```

Comment:
Are the Release Notes for 1.8 supposed to be hidden?  This include is commented out.